python/gps/algorithm/cost/cost_fk_blocktouch.py

```diff
@@ -0,0 +1,56 @@
+# TODO: This cost function is hard-coded and not easy to generalize.
+import copy
+
+import numpy as np
+
+from gps.algorithm.cost.config import COST_FK
+from gps.algorithm.cost.cost import Cost
+from gps.algorithm.cost.cost_utils import get_ramp_multiplier
+from gps.proto.gps_pb2 import JOINT_ANGLES, END_EFFECTOR_POINTS, \
+        END_EFFECTOR_POINT_JACOBIANS
+
+
+class CostFKBlock(Cost):
```

Comment:
Can you add a brief description of this cost function? (like the binary cost description)